### ISSUE TITLE
Add citation.cff

### ISSUE THREAD

**Issue by samumantha** [coderefinery/reproducible-research#240]
Could be prepared even though future steps are not resolved yet?

**Comment by bast** [comment]
Yes. I also believe that we will get the rest in place before workshop.

**Comment by samumantha** [comment]
-> #245

**Comment by samumantha** [comment]
Dear contributors of this lesson, to the best of my abilities I have tried to identify "big enough" contributions (based on commits) to this lesson to give you the title of being an author of this lesson. There is no strict guidelines and if you feel that you would either like to be removed from authors or be added to authors, now is your time to speak up!

Everyone mentioned in the CITATION.cff file will also be author of the lesson in Zenodo.
No further action required if you are ok with things as they are.

Pinging people who would currently be listed as authors: @wikfeldt , @bast , @rkdarst  , @johanhellsvik , @samumantha , @MatiasJJ , @eglerean and @Vathasav .

And pinging also everyone with minor contributions to speak up, if you would like to be listed as an author: @tpfau, @ruokolt, @code4yonglei, @Sparrow0hawk, @PatricHolmvall, @ThomasA, @korbinib, @matuskalas, @blankdots, @gregordecristoforo 

And to everyone: You may also have contributed to this lesson in other ways than a git commit, eg being part of discussions or similar. This would also be a contribution, so do not hesitate to ask to be added to authors, or also suggest others not in any of the list to be added to authors. 

Thank you!

**Comment by samumantha** [comment]
Thank you all for your contributions, the PR is now ready and once merged this issue will be closed.

**Comment by code4yonglei** [comment]
I am glad to be one of the authors. What should I do at the current stage? @samumantha

**Comment by samumantha** [comment]
Please send a pull request adding yourself to the list of authors in the citation.cff file. Thank you!
